File: Assignment No.37/Program2.java
/*Problem Statement :Write a Program which accept one number and whether that 5 bit and 18 bit is on or off.
Input :  
Output : 
*/
import java.util.*;

class Program2
{
	public static boolean ChkBit(int iNo)
	{
		int iMask=0x00020010,iResult=0;

		iResult=iNo & iMask;
		if(iResult == iMask)
		{
			return true;
		}
		else
		{
			return false;
		}
			 
	}
	public static void main(String args[])
	{
		int iNo = 0;
		Boolean bret=false;

		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter the Number :");
		iNo = sobj.nextInt();
		
		bret = ChkBit(iNo);
		if(bret == true)
		{
			System.out.println("15 bit & 5 bit is on");
		}
		else
		{
			System.out.println("15 bit & 5 bit is off");
		}
		
	}
}	
		

File: Assignment No 5/Program2.c
//Write a Program which accept name from user which accept one nummber from user and check whether that numberis greater that100 or not..
//Input : 101
//Output : Greater
//Input : 39
//Output : smaller

#include<stdio.h>
#include<stdbool.h>

typedef int bool

#define TRUE 1
#define FALSE 0
bool ChkGreater(int iNo)
{
	if(iNo>100)
	{
		return TRUE;
	}
	else
	{
		return FALSE;
	}
}
int main()
{
	int iValue=0;
	bool bRet=FALSE;

	printf("Please enter number :\n");
	scanf("%d",&iValue);
	bRet=ChkGreater(iValue);
	if(bRet==true)
	{
		printf("Greater");
	}
	else
	{
		printf("smaller");
	}
	return 0;
}